packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -192,6 +193,10 @@ const websiteEntityPage = (
     <EntityLayout.Route path="/docs" title="Docs">
       {techdocsContent}
     </EntityLayout.Route>
+
```

Comment:
Do you think we should add this to the serviceEntityPage and groupPage to show off more places that it can be implemented at? I can't think of any services that we might demo, but there will be our Janus Group in the app and we can have the GitHub issues there as well.